src/gluonts/model/gp_forecaster/_estimator.py

```diff
@@ -104,15 +104,15 @@ def __init__(
         context_length: Optional[int] = None,
         kernel_output: KernelOutput = RBFKernelOutput(),
         params_scaling: bool = True,
-        float_type: DType = np.float64,
+        dtype: DType = np.float64,
         max_iter_jitter: int = 10,
         jitter_method: str = "iter",
         sample_noise: bool = True,
         time_features: Optional[List[TimeFeature]] = None,
         num_parallel_samples: int = 100,
     ) -> None:
-        self.float_type = float_type
-        super().__init__(trainer=trainer, float_type=self.float_type)
+        self.float_type = dtype
+        super().__init__(trainer=trainer, dtype=self.float_type)
```

Comment:
`self.dtype`?

Reply:
Ideally I should replace `float_type` with `dtype` here and remove this line (why did we have this line here anyway? this is not needed here); but refactoring the name triggers lot of changes on the `gp_forecaster` so postponing it.